includes/admin/class-wc-admin-importers.php

```diff
@@ -31,6 +31,8 @@ public function __construct() {
 		add_action( 'admin_menu', array( $this, 'add_to_menus' ) );
 		add_action( 'admin_init', array( $this, 'register_importers' ) );
 		add_action( 'admin_head', array( $this, 'hide_from_menus' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_scripts' ) );
+		add_action( 'wp_ajax_woocommerce_do_ajax_product_import', array( $this, 'do_ajax_product_import' ) );
```

Comment:
To keep it all in one place TBH. It's easier to develop that way, and if something disables our classes the functions won't stick around.